lib/project/pro_motion_form/form_screen.rb

```diff
@@ -0,0 +1,8 @@
+module ProMotion
+  class FormScreen
+    # Defining this here is critical for PM::FormScreens to not fire delegate
+    # methods twice. This class must be defined even if we aren't using
+    # the ProMotion-form gem so that our pm_handles_delegates? method
+    # works properly for those people who _do_ use PM::FormScreen
+  end
```

Comment:
Personally if we are going this direction I'd rather define it in `/ext` and just open the class and define if it should handle the delegate (like my other examples in #91)

In other words, here we would want to add the `delegates_handled?` method returning true (or some other named method) and we check if it responds to, and this value is true

what does everyone else think?

then we dont have this method that continues to grow `is_a?` conditions for every time of controller that may already cover the call to `view_did_load`